lib/k_chart_widget.dart

```diff
@@ -312,8 +312,7 @@ class _KChartWidgetState extends State<KChartWidget>
       color = widget.chartColors.infoWindowUpColor;
     else if (info.startsWith("-"))
       color = widget.chartColors.infoWindowDnColor;
-    else
-      color = widget.chartColors.infoWindowNormalColor;
```

Comment:
it's already set as Color color = widget.chartColors.infoWindowNormalColor; before if-else if